modular_skyrat/master_files/code/modules/clothing/under/jobs/civilian/civilian.dm

```diff
@@ -47,3 +47,101 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	dying_key = DYE_REGISTRY_JUMPSKIRT
 	female_sprite_flags = FEMALE_UNIFORM_TOP_ONLY
+
+/*
+*     BARTENDER
+*/
+
+/obj/item/clothing/under/civilian/bartender/sleeky
+    name = "Sleek Bartender Suit"
+    desc = "A Post-Postmodern take on a Retrofuturistic garment, how innovative!"
+    icon = 'modular_skyrat/master_files/icons/obj/clothing/under/civilian.dmi'
+    icon_state = "sleekbartender"
+    worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/civilian.dmi'
+    can_adjust = FALSE
+
+/obj/item/clothing/under/civilian/bartender/sleeky/skirt
+    name = "Sleek Bartender Skirt"
+    desc = "A Post-Postmodern take on a Retrofuturistic garment, how innovative!"
+    icon = 'modular_skyrat/master_files/icons/obj/clothing/under/civilian.dmi'
+    icon_state = "sleekbartender_skirt"
+    worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/civilian.dmi'
```

Comment:
this second's the above comment, when this becomes a subtype of /skyrat, you won't have to re-define icon, worn_icon, or can_adjust. Moreover, as a subtype of /sleeky, you won't need to re-define desc, as it's a duplicate of the previous type's desc